sw/device/tests/sim_dv/sram_ctrl_execution_test_main.c

```diff
@@ -45,9 +45,10 @@ static const uint32_t kRamEndAddr = TOP_EARLGREY_SRAM_CTRL_MAIN_RAM_BASE_ADDR +
  *
  * x = OTP_CTRL_PARAM_EN_SRAM_IFETCH_OFFSET (1728)
  * y = OTP_CTRL_PARAM_HW_CFG_OFFSET (1664)
- * IFETCH_OFFSET = (x - y) / 8 = 8
+ * IFETCH_OFFSET = (x - y) = 64
  */
-static const uint32_t kOtpIfetchHwRelativeOffset = 8;
+static const uint32_t kOtpIfetchHwRelativeOffset = 64;
```

Comment:
You could use the register offsets defined in the (autogenerated) `otp_ctrl_regs.h` and rewrite this as:

```c
static const uint32_t kOtpIfetchHwRelativeOffset = OTP_CTRL_PARAM_EN_SRAM_IFETCH_OFFSET - OTP_CTRL_PARAM_HW_CFG_OFFSET;
```
and maybe add a `static_assert(kOtpIfetchHwRelativeOffset == 64)` that checks the computed offset is what you expect it to be, in case the OTP layout ever changes?